src/CSnakes.Runtime/CPython/Buffer.cs

```diff
@@ -0,0 +1,58 @@
+﻿using CSnakes.Runtime.Python;
+using System.Runtime.InteropServices;
+
+namespace CSnakes.Runtime.CPython;
+internal unsafe partial class CPythonAPI
+{
+    [Flags]
+    private enum PyBUF
+    {
+        Simple = 0,
+        Writable = 0x1,
+        Format = 0x4,
+        ND = 0x8,
+        Strides = 0x10 | ND,
+        CContiguous = 0x20 | Strides,
+        FContiguous = 0x40 | Strides,
+        AnyContiguous = 0x80 | Strides,
+    }
+
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct Py_buffer
+    {
+        public IntPtr buf;
+        public IntPtr obj;
```

Comment:
any reason for these not be a `nint`?
```suggestion
        public nint buf;
        public nint obj;
```